controller_manager/include/controller_manager/controller_manager.hpp

```diff
@@ -220,15 +220,6 @@ class ControllerManager : public rclcpp::Node
   CONTROLLER_MANAGER_PUBLIC
   unsigned int get_update_rate() const;
 
-  /// Deletes all async controllers and components.
-  /**
-   * Needed to join the threads immediately after the control loop is ended
-   * to avoid unnecessary iterations. Otherwise
-   * the threads will be joined only when the controller manager gets destroyed.
-   */
-  CONTROLLER_MANAGER_PUBLIC
```

Comment:
could you please try building this branch against webots too?

Reply:
Sure!

Reply:
Done. I checked it and it is compiling without issues.

Reply:
Moreover, they are also not using it.


https://github.com/search?q=repo%3Acyberbotics%2Fwebots_ros2+shutdown_async_controllers_and_components&type=code